### PR TITLE
Add GitHub issue template

### PR DESCRIPTION
**Pull request by tamird**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/5713)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Please answer these questions *for each node in your cluster* before
```
Comment (petermattis) [inline .github/ISSUE_TEMPLATE.md]: This only applies to the first 3 questions, correct?

**Comment by petermattis** [comment]
LGTM

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (tamird) [inline .github/ISSUE_TEMPLATE.md]: Good point, updated.

Comment (petermattis) [inline .github/ISSUE_TEMPLATE.md]: Yep, that's better.

**Pull request merged**